sdks/python/apache_beam/io/avroio.py

```diff
@@ -586,30 +591,32 @@ def __init__(self,
     self._schema = schema
     self._codec = codec
 
+  def display_data(self):
+    res = super(_BaseAvroSink, self).display_data()
+    res['codec'] = str(self._codec)
+    res['schema'] = str(self._schema)
+    return res
+
+
+class _AvroSink(_BaseAvroSink):
```

Comment:
We probably want to throw an error with a message that Avro is not available on Python 3 right now, when this class is being used on Python 3. I think Juta already added something similar in another module, so you can check with her how to handle this.